test/browser_chrome_test.dart

```diff
@@ -0,0 +1,186 @@
+@TestOn('chrome')
```

Comment:
I imagine that the two files for Firefox and Chrome are quite similar.

Can we make this a single file.

Perhaps we can make `detectedRuntime` to handle different browsers differently.

If we only have a single file it's easier to spot the difference.